util/carvelhelpers/kbld.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package carvelhelpers implements wrapper functions to use carvel tooling
+package carvelhelpers
```

Comment:
These don't belong in the `util` module (yet). If you think it does, please provide justification as to how it might be used by multiple modules. Otherwise, just move this to `cluster-essentials/pkg`. 

Functions/methods in the `util` module need to have much more friendlier APIs (these are somewhat hard to understand) and have test cases.